stress/src/logs.rs

```diff
@@ -51,13 +57,26 @@ impl LogProcessor for MockLogProcessor {
     fn shutdown(&self) -> OTelSdkResult {
         Ok(())
     }
+
+    fn event_enabled(
+        &self,
+        _level: opentelemetry::logs::Severity,
+        _target: &str,
+        _name: Option<&str>,
+    ) -> bool {
+        self.enabled
+    }
 }
 
 fn main() {
+    // change this to false to test the throughput when enabled is false.
+    let enabled = true;
```

Comment:
nit - let it pass as command line argument.

Reply:
yes I mentioned that in the description too.